### ISSUE TITLE
linting

### ISSUE THREAD

**Issue by djdanielsson** [ansible/ansible-content-actions#23]
is there a reason this doesn't just call the already created ansible/ansible-lint action?

**Comment by ssbarnea** [comment]
Shortly, we could do this.

Long story: linting is likely one of the most easy actions to implement. We goal of content actions is far bigger and it also wants to fully cover running the same tests from CI locally, so when a particular job fails, you will know how to reproduce the failure locally.

I plan to do some major architectural (breaking) changes to the current implementation in the following couple of months. Mainly is about learning from the experience made using https://github.com/coactions/dynamic-matrix and make something similar that makes use of tox-ansible in order to generate the dynamic matrix and the entire workflow. Lint will be just one step there.

Basically run-ansible-lint action is probably the easiest thing to implement for a new collection. But once you want to take it to the next level and have a full testing and publishing pipeline, you might just switch to use the content actions. If they use the run-ansible-lint action or not, would be just an implementation detail.

**Comment by djdanielsson** [comment]
I was thinking for consistency sake wouldn't it be best if this calls that to do the linting that way any lint action bugs are reported and fixed in one location? Also it ensures that if someone started with linting this action just builds onto that and still functions (lint wise) exactly the same.

**Comment by ssbarnea** [comment]
I will be closing this with the note that while ansible-lint will continue to be available as a standalone action, we will expose lint as a job from this repository too, but it will one of many others.
